tensorboard/webapp/metrics/views/card_renderer/scalar_card_fob_controller.ts

```diff
@@ -99,6 +100,10 @@ export class ScalarCardFobController {
     );
   }
 
+  onPrespectiveStepChanged(step: number | null) {
```

Comment:
Ok, I am a little late here but, this should be onProspectiveStepChanged right? Note the o in Prospective.

Reply:
ah, thanks for catching this. Yes that's typo and should be fixed in next PR